README.md

```diff
@@ -1,2 +1,248 @@
-# hello-rails-back-end
-Here provided Api for the back-end.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello Rails Back-end</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello Rails Back-end ](#ruby-on-rails-group-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello rails back-end<a name="ruby-on-rails-group-project"></a>
+
+**Hello rials Back-end** 
+Hello rails Back-end is an application that rails as an API.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> So you want to lear more about the tech we have used for this project?
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+<summary>Front-end</summary>
+  <ul>
+    <li><a href="#">React</a></li>
+  </ul>
```

Comment:
- [ ] Please make sure to attach the link to the front-end repo here in the back-end repo's README file as well. 🙏